cmd/testgrid-config-generator/main.go

```diff
@@ -223,12 +225,23 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatalf("Failed to load Prow jobs %s", o.prowJobConfigDir)
 	}
+
+	// read the list of jobs from the allow list along with its release-type
+	data, err := ioutil.ReadFile(o.jobsAllowListFile)
+	if err != nil {
+		logrus.WithError(err).Fatalf("could not read allow-list at %s", o.jobsAllowListFile)
+	}
+
+	var allowList map[string]string
+	if err := yaml.Unmarshal(data, &allowList); err != nil {
+		logrus.WithError(err).Fatalf("could not unmarshal allow-list at %s", o.jobsAllowListFile)
+	}
+
 	for _, p := range jobConfig.Periodics {
 		name := p.Name
 		calculateDays := len(p.Cron) > 0 || len(p.Interval) > 0
-
 		var dashboardType string
-		switch label := p.Labels["ci.openshift.io/release-type"]; label {
```

Comment:
We should support both cases while we migrate to the allow-list and remove the annotations.

Reply:
I think I have added all the cases where there is the release-type annotation to the _allow-list.yaml file. Unless you are expecting users to continue adding these annotations to the jobs configs directly, we should be ok I think. Is there anything I missed?

Reply:
This would also force users to start using the _allow-list.yaml file instead of continuing with modifying the jobs config